WLC/code_executor/executor.py

```diff
@@ -52,6 +55,30 @@ def execute_code(self, code):
 
         return result, error
 
+    def execute_hacker_rank(self, code, test_key):
```

Comment:
One small thing here, rename this to `execute_tests` or something? Removing hackerrank completely.